microsoft-edge/webview2/samples/wv2deploymentwixcustomactionsample.md

```diff
@@ -138,7 +138,7 @@ If you want the app to download the Evergreen WebView2 Runtime Bootstrapper (`Mi
 
 1. Comment out other `<Binary>` and `<CustomAction>` elements under `Step 4`.
```

Comment:
in edge-developer\microsoft-edge\webview2\how-to\winrt-from-js.md
reword from 
To ensure that the JavaScript that sets all this up is **executed** before anything else, you can either add the above statement to your JavaScript, or you can tell WebView2 to inject the above statement for you before running any other script, by using the `CoreWebView2.AddScriptToExecuteOnDocumentCreatedAsync` method.

Reply:
fixed locally